doc/motr-in-prose.md

```diff
@@ -149,9 +149,8 @@ Motr used to be called SCIF and H3S in the earlier documents.
 Lustre is a cluster file-system, designed in late 90s early 00s by Peter Braam 
 at CMU. Lustre was developed by ClusterFS, which was acquired by Sun 
 Microsystems, which was acquired by Oracle, Lustre IP was then acquired by 
-Xyratex, which is in process of being acquired by Seagate—file systems habent 
-sua fata. Lustre is highly successful in the HPC space, running on 7 to 9 
-supercomputers from Top10 list. 
+Xyratex, which was acquired by Seagate. Lustre is highly successful in the
+HPC space, running on 7 to 9 supercomputers from Top10 list.
```

Comment:
Why was the rest of the sentence truncated?

Reply:
Latin, not standard English, is difficult to understand.
But if you think it's better to keep them, we can revert it.

Reply:
Truncated text added back